smart-contract-verifier/smart-contract-verifier-http/src/settings.rs

```diff
@@ -219,6 +225,12 @@ impl Default for CompilersSettings {
     }
 }
 
+#[derive(Deserialize, Clone, PartialEq, Eq, Debug)]
+#[serde(deny_unknown_fields)]
+pub struct Extensions {
+    pub sig_provider: sig_provider_extension::Config,
```

Comment:
I suggest to make each extension optional, while initializing it only of corresponding setting was given. That is, to make `extension: Extension` field not optional for the corresponding structs, but make each extension optional in `Extensions`. So that when new extensions would be added, we could enable only some of them